crates/router/src/core/payment_methods/cards.rs

```diff
@@ -1184,6 +1192,19 @@ fn filter_pm_based_on_config<'a>(
         .unwrap_or(true)
 }
 
+fn filter_for_capture_method(
```

Comment:
This function could either use a better name or have a brief documentation as to what is does.